plantisc2/tasks/app.yml

```diff
@@ -6,3 +6,14 @@
     version: remove-archive
   become: yes
   become_user: www-data
+
+- name: Copy sql Script
+  template:
+    src: tissue.sql
```

Comment:
If a file is a template, it should end with .j2 (Jinja template). 
Suggest changing the name of the file from `tissue.sql` to `tissue.sql.j2`